### ISSUE TITLE
Implement basic Sass outline from Pillars

### ISSUE THREAD

**Issue by jonlay** [wearehanno/riggings#16]
We have a nice, clear Sass structure [defined in Pillars](https://github.com/wearehanno/pillars/blob/master/sass/example.scss) but we don't have this added by default to Riggings.

Could we add a few basic Sass files (without any complex style rules) into Riggings to make it easier to meet the Pillars conventions and structure our Sass more consistently on new projects?

**Comment by sergeitatarinov** [comment]
Yup, need to update Pillars as well with new learnings etc. A little bit outdated now.
